servers/rendering/rasterizer_rd/shaders/canvas_uniforms_inc.glsl

```diff
@@ -84,16 +72,6 @@ layout(set = 2, binding = 0, std140) uniform CanvasData {
 }
 canvas_data;
 
-layout(set = 2, binding = 1) uniform textureBuffer skeleton_buffer;
-
-layout(set = 2, binding = 2, std140) uniform SkeletonData {
-	mat4 skeleton_transform; //in world coordinates
-	mat4 skeleton_transform_inverse;
```

Comment:
removed here, but not re-added

Reply:
Ah yeah is fine, when I add skeletons again I will do it different I think.